src/include/hardware.h

```diff
@@ -168,6 +168,8 @@
 
 #define FLD_ISSET_ATOMIC_16(field, mask) (__wt_atomic_load16(&(field)) & (uint16_t)(mask))
 
+#define FLD_ISSET_ATOMIC_32(field, mask) (__wt_atomic_load32(&(field)) & (uint32_t)(mask))
```

Comment:
When reviewing this PR there's only one real change: I've added the new ATOMIC_32 definitions in hardware.h, and then all the other diffs (bar my comment in `rollback_to_stable.h`) are a find and replace to change `F_*` to `F_*_ATOMIC_32`